hs_app_timeseries/tests/views/test_create_timeseries_resource.py

```diff
@@ -0,0 +1,141 @@
+from django.contrib.messages.storage.fallback import FallbackStorage
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from hs_core import hydroshare
+from hs_core.models import BaseResource, ResourceFile
+from hs_core.views import create_resource
+from hs_core.testing import MockIRODSTestCaseMixin
+
+
+class TestCreateResourceViewFunctions(MockIRODSTestCaseMixin, TestCase):
+    def setUp(self):
+        super(TestCreateResourceViewFunctions, self).setUp()
+        self.group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        self.username = 'john'
+        self.password = 'jhmypassword'
+        self.user = hydroshare.create_account(
+            'john@gmail.com',
+            username=self.username,
+            first_name='John',
+            last_name='Clarson',
+            superuser=False,
+            password=self.password,
+            groups=[]
+        )
+
+        self.factory = RequestFactory()
+
+        self.temp_dir = tempfile.mkdtemp()
+
+        self.odm2_sqlite_file_name = 'ODM2_Multi_Site_One_Variable.sqlite'
+        self.odm2_sqlite_file = 'hs_app_timeseries/tests/{}'.format(self.odm2_sqlite_file_name)
+        target_temp_sqlite_file = os.path.join(self.temp_dir, self.odm2_sqlite_file_name)
+        shutil.copy(self.odm2_sqlite_file, target_temp_sqlite_file)
+        self.odm2_sqlite_file_obj = open(target_temp_sqlite_file, 'r')
+
+        self.odm2_sqlite_invalid_file_name = 'ODM2_invalid.sqlite'
+        self.odm2_sqlite_invalid_file = 'hs_app_timeseries/tests/{}'.format(
+            self.odm2_sqlite_invalid_file_name)
+        target_temp_sqlite_invalid_file = os.path.join(self.temp_dir,
+                                                       self.odm2_sqlite_invalid_file_name)
+        shutil.copy(self.odm2_sqlite_invalid_file, target_temp_sqlite_invalid_file)
+        self.odm2_sqlite_invalid_file_obj = open(target_temp_sqlite_invalid_file, 'r')
+
+    def tearDown(self):
+        super(TestCreateResourceViewFunctions, self).tearDown()
+        if os.path.exists(self.temp_dir):
+            shutil.rmtree(self.temp_dir)
+
+    def test_create_resource(self):
+        # here we are testing the create_resource view function
+
+        # test with no file upload
+        post_data = {'resource-type': 'TimeSeriesResource',
+                     'title': 'Test Time Series Resource Creation',
+                     'irods_federated': 'true'
+                     }
+        url = reverse('create_resource')
+        request = self.factory.post(url, data=post_data)
+        request.user = self.user
+        self._set_request_message_attributes(request)
+        self._add_session_to_request(request)
+
+        response = create_resource(request)
+        self.assertEqual(response.status_code, status.HTTP_302_FOUND)
+        res_id = response.url.split('/')[2]
+        self.assertEqual(BaseResource.objects.filter(short_id=res_id).exists(), True)
```

Comment:
change to to assertTrue(). You can search through all occurrences of such and make changes to all of them